src/Resources/ShippingResource.php

```diff
@@ -46,11 +46,11 @@ public function getShipment(string $shipment_id): GetShipmentResponse
         return new GetShipmentResponse($response);
     }
 
-    public function cancelShipment(string $shipment_id): CancelShipmentResponse
+    public function cancelShipment(string $shipment_id): BaseResponse
     {
         $response = $this->http->post($this->endpoint . self::BASE_PATH . 'shipments/' . $shipment_id . '/cancel', []);
 
-        return new CancelShipmentResponse($response);
+        return new BaseResponse($response);
```

Comment:
You don't need to remove the CancelShipmentResponse, it just doesn't need to contain the ErrorListSchema